extension/content.css

```diff
@@ -1,13 +1,11 @@
-.highlighted {
-	color: #017f92;
-}
-
 mark {
-	background-color:#c9daf8 !important;
-	background: #c9daf8 !important;
-	transition: all 0.5s ease 0.5s !important;
 }
-.marked {
-	background: #c9daf8 !important;
+.markedOne {
```

Comment:
instead of one,two, prefer to pick memorable words so readers know which of the 3 is at a glance.

Reply:
Agree [fire emoji]